### DISCUSSION TITLE
Connection OK with Trino but I have after the same error of connection with all the SQL queries?

### DISCUSSION THREAD

**Discussion by BorderCloud** [ontop/ontop#762] [Q&A] [answered]
Hello,

I have done the tutorial and now I test with Trino. The test connection is OK.

<img width="443" alt="trino_jdbc" src="https://github.com/ontop/ontop-website/assets/352813/c553c2c2-dc5d-4260-b91c-b6b498ddb80d">
<img width="473" alt="ontop_trino" src="https://github.com/ontop/ontop-website/assets/352813/a8f28400-0aae-423e-9bb4-ecb63bb63b99">

But all the queries (with from mongo.hello.world also) send the same error:
<img width="953" alt="error_trino_mongo_in_ontop" src="https://github.com/ontop/ontop-website/assets/352813/5401f236-8ba4-4547-afe1-2550871d5322">

In logs of Protégé/Ontop/Trino/Mongo, there is no error (I have enables log debug in Protégé). 
Do you have an idea to see the logs about this error?

Thanks.

**Comment by BorderCloud** [comment, marked as answer]
The interface to test the connection responds always "OK" but it's wrong. Probably a bug.
I tested several connection url and finally, I found.
Thanks

**Reply by bcogrel** [reply]
Thanks for reporting. I opened an issue to keep track of it: https://github.com/ontop/ontop/issues/766
